### PR TITLE
📝 Changed FirebaseWriteRequest id to nullable string type

### PR DESCRIPTION
**Pull request by Abhishek-Acme**

### CODE REVIEW

File: lib/src/defaults/providers/firebase/firebase_requests.dart [hunk 1]
```diff
@@ -29,7 +29,7 @@ class FirebaseReadIdRequest extends FirebaseRequest {
 }
 
 class FirebaseWriteRequest extends FirebaseRequest {
-  final String id;
+  final String? id;
   FirebaseWriteRequest({required String path, required this.id})
```
Comment (sarbagya-acme) [inline lib/src/defaults/providers/firebase/firebase_requests.dart]: Shouldn't we make it optional too ?

**Review by sarbagya-acme — CHANGES_REQUESTED** [review]

File: lib/src/defaults/providers/firebase/firebase_requests.dart (hunk 1 above)
Comment (Abhishek-Acme) [reply lib/src/defaults/providers/firebase/firebase_requests.dart]: Yes, let's do it as well.

**Comment by sarbagya-acme** [comment]
@Abhishek-Acme Can you make the check pass ?

**Comment by sarbagya-acme** [comment]
Looks like a false positive coverage check.

**Pull request merged**